test/TestConfig/scripts/build_test.xml

```diff
@@ -67,11 +67,11 @@
 	</if>
 	<echo> build.list is ${build.list} </echo>
 
-	<target name="build" depends="stage_test_material" description="using ${JAVA_VERSION} compile the source ">
+	<target name="build" depends="stage_test_material" description="using ${JDK_VERSION} compile the source ">
```

Comment:
Using java ${JDK_VERSION} to compile the source

Reply:
updated